frontend/fingertips-frontend/components/organisms/LineChartTable/LineChartTable.test.tsx

```diff
@@ -1,74 +1,63 @@
 import { expect } from '@jest/globals';
 import { LineChartTable } from '@/components/organisms/LineChartTable/index';
+import {
+  LIGHT_GREY,
+  LineChartTableHeadingEnum,
+} from '@/lib/chartHelpers/chartHelpers';
+import { mockHealthData } from '@/mock/data/healthdata';
 
-const mockHeadings = [
-  'Area Code',
-  'Year',
-  'Value',
-  'Count',
-  'LowerCi',
-  'UpperCi',
-];
+describe('Line chart table suite', () => {
+  const MOCK_HEALTH_DATA = mockHealthData['1'][0];
+  const CELLS_PER_ROW = 7;
 
-const mockData = [
-  {
-    areaCode: 'A1425',
-    healthData: [
-      {
-        count: 389,
-        lowerCi: 441.69151,
-        upperCi: 578.32766,
-        value: 278.29134,
-        year: 2006,
-        sex: 'Persons',
-        ageBand: 'All',
-      },
-      {
-        count: 267,
-        lowerCi: 441.69151,
-        upperCi: 578.32766,
-        value: 703.420759,
-        year: 2004,
-        sex: 'Persons',
-        ageBand: 'All',
-      },
-    ],
-  },
-  {
-    areaCode: 'A1426',
-    healthData: [
-      {
-        count: 389,
-        lowerCi: 441.69151,
-        upperCi: 578.32766,
-        value: 278.29134,
-        year: 2006,
-        sex: 'Persons',
-        ageBand: 'All',
-      },
-      {
-        count: 267,
-        lowerCi: 441.69151,
-        upperCi: 578.32766,
-        value: 703.420759,
-        year: 2004,
-        sex: 'Persons',
-        ageBand: 'All',
-      },
-    ],
-  },
-];
+  it('snapshot test - should match snapshot', () => {
+    const container = render(<LineChartTable data={MOCK_HEALTH_DATA} />);
+    expect(container.asFragment()).toMatchSnapshot();
+  });
 
-it('snapshot test - should match snapshot', () => {
-  const container = render(
-    <LineChartTable data={mockData} headings={mockHeadings} />
-  );
-  expect(container.asFragment()).toMatchSnapshot();
-});
+  it('should render the LineChartTable component', () => {
+    render(<LineChartTable data={MOCK_HEALTH_DATA} />);
+    const lineChart = screen.getByTestId('lineChartTable-component');
+    expect(lineChart).toBeInTheDocument();
+  });
+
+  it('should render expected elements', () => {
+    render(<LineChartTable data={MOCK_HEALTH_DATA} />);
+
+    expect(screen.getByRole('table')).toBeInTheDocument();
+    expect(screen.getAllByRole('columnheader')[0]).toHaveTextContent(
+      MOCK_HEALTH_DATA.areaCode
+    );
+    expect(screen.getByText(/95% confidence limits/i)).toBeInTheDocument();
+    expect(screen.getByText(/England/i)).toBeInTheDocument();
+    expect(screen.getAllByRole('cell')).toHaveLength(
+      MOCK_HEALTH_DATA.healthData.length * CELLS_PER_ROW
+    );
+    Object.values(LineChartTableHeadingEnum).forEach((heading, index) =>
+      expect(
+        screen.getByTestId(`header-${heading}-${index}`)
+      ).toBeInTheDocument()
+    );
+    screen.getAllByTestId('grey-table-cell').forEach((greyCell) => {
+      expect(greyCell).toHaveStyle(`background-color: ${LIGHT_GREY}`);
+    });
```

Comment:
I think let's not bother with checking the styling as part of the unit tests. Unless we have some specific logic to do a specific style for a specific condition. 

Will use the snapshot tests for that purpose. Otherwise the tests will start to get too much.

Reply:
Looking at the code now. I can see why you have done it. Can you do a seperate tests to say that the benchmark column should be in grey or something along those lines instead.